include/common.h

```diff
@@ -28,51 +43,89 @@ typedef struct {
     /* 0x24 */ f32 unk24;
     /* 0x28 */ f32 unk28;
     /* 0x2C */ f32 unk2C;
-    /* 0x30 */ u32 unk30;
+    /* 0x30 */ u32 pad30;
     /* 0x34 */ f32 unk34;
     /* 0x38 */ f32 scale;
-    /* 0x3C */ u32 unk3C;
-    /* 0x40 */ u32 unk40;
+    /* 0x3C */ Unk80132ED0 *unk3C;
+    /* 0x40 */ s32 pad40;
     /* 0x44 */ f32 rotation[3];
     /* 0x50 */ f32 unk50;
     /* 0x54 */ f32 unk54;
-    /* 0x58 */  f32 unk58;
-    /* 0x5C */  u8 unk5C;
-    /* 0x5D */ u8 unk5D;
-    /* 0x5E */ u8 unk5E;
+    /* 0x58 */ f32 unk58;
+    /* 0x5C */ u8 pad5C;
+    /* 0x5D */ u8 pad5D;
+    /* 0x5E */ u8 pad5E;
     /* 0x5F */ u8 unk5F;
-    /* 0x60 */ f32 unk60;
-    /* 0x64 */ u32 unk64_20 : 12;
-    /* 0x64 */ u32 unk64_19 : 1;
-    /* 0x64 */ u32 unk64_18 : 1;
-    /* 0x64 */ u32 unk64_17 : 1;
-    /* 0x64 */ u32 unk64_16 : 1;
-    /* 0x68 */ u32 unk68;
-    /* 0x68 */ u32 unk6C_24 : 7;
-    /* 0x68 */ u32 unk6C_16 : 9;
-    /* 0x68 */ u32 unk6C_8 : 7;
-    /* 0x68 */ u32 unk6C_0 : 9;
-    /* 0x70 */ u16 unk70;
-    /* 0x72 */ u16 unk72;
-    /* 0x74 */ u32 unk74_31 : 1;
-    /* 0x74 */ u32 unk74_30 : 1;
-    /* 0x74 */ u32 unk74_16 : 14;
-    /* 0x74 */ u32 unk74_7 : 9;
+    /* 0x60 */ s32 pad60;
+    /* 0x64 */ union {
+        u32 flags;
+        u8 bytes[4];
+        struct {
+            u32 pad64_20 : 12;
+            u32 unk64_19 : 1; // flags & 0x80000
+            u32 pad64_17 : 2;
+            u32 unk64_16 : 1;
+            u32 pad64_0 : 16;
+        } bits;
+    } unk64;
+    /* 0x68 */ s32 pad68;
+    /* 0x6C */ u32 pad6C_9 : 23;
+    /* 0x6C */ u32 unk6C_0 : 9; // if accessed directly: unk6C & 0x1FF, if assigned: ((u16)valueToAssign & 0x1FF) | (unk6E & 0xFE00);
+    /* 0x70 */ u32 pad70_16: 16;
+    /* 0x70 */ u32 unk70_10: 6; // unk72 >> 10
+    /* 0x70 */ u32 pad70_0: 10;
+    /* 0x74 */ union {
```

Comment:
These changes to unk74 aren't needed, as everything in this PR can be matched without using `unk74.flags` or `unk74.bytes`.